doc/tools/make_rst.py

```diff
@@ -1053,6 +1054,13 @@ def rstize_text(text, state):  # type: (str, State) -> str
                 # Strip newline if the tag was alone on one
                 if pre_text[-1] == "\n":
                     pre_text = pre_text[:-1]
+            elif cmd == "/note" or cmd == "/warning":
+                tag_text = ""
+                tag_depth -= 1
+                inside_note_warning = False
+                # Strip newline if the tag was alone on one
```

Comment:
I see that it's copied from the above sample, but I don't get that comment. It feels like it's cut off, or it's a typo. Do you know what it actually supposed to mean and can you fix it (here and above too)?

Reply:
I copied that part without understanding what it was really useful for.